2021-11-09 22:39:49,225 INFO    MainThread:33613 [wandb_setup.py:_flush():71] setting env: {}
2021-11-09 22:39:49,225 INFO    MainThread:33613 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-09 22:39:49,225 INFO    MainThread:33613 [wandb_init.py:_log_setup():357] Logging user logs to /home/younesbelkada/Travail/MVA/Recvis/Assignment3/recvis21_a3/wandb/run-20211109_223949-1dcv56xc/logs/debug.log
2021-11-09 22:39:49,226 INFO    MainThread:33613 [wandb_init.py:_log_setup():358] Logging internal logs to /home/younesbelkada/Travail/MVA/Recvis/Assignment3/recvis21_a3/wandb/run-20211109_223949-1dcv56xc/logs/debug-internal.log
2021-11-09 22:39:49,226 INFO    MainThread:33613 [wandb_init.py:init():390] calling init triggers
2021-11-09 22:39:49,226 INFO    MainThread:33613 [wandb_init.py:init():397] wandb.init called with sweep_config: {}
config: {}
2021-11-09 22:39:49,227 INFO    MainThread:33613 [wandb_init.py:init():435] starting backend
2021-11-09 22:39:49,227 INFO    MainThread:33613 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-09 22:39:49,246 INFO    MainThread:33613 [backend.py:ensure_launched():199] starting backend process...
2021-11-09 22:39:49,254 INFO    MainThread:33613 [backend.py:ensure_launched():205] started backend process with pid: 33631
2021-11-09 22:39:49,256 INFO    MainThread:33613 [wandb_init.py:init():444] backend started and connected
2021-11-09 22:39:49,260 INFO    MainThread:33613 [wandb_init.py:init():503] updated telemetry
2021-11-09 22:39:49,280 INFO    MainThread:33613 [wandb_init.py:init():533] communicating current version
2021-11-09 22:39:50,072 INFO    MainThread:33613 [wandb_init.py:init():538] got version response 
2021-11-09 22:39:50,072 INFO    MainThread:33613 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-11-09 22:39:50,396 INFO    MainThread:33613 [wandb_init.py:init():576] starting run threads in backend
2021-11-09 22:39:50,432 INFO    MainThread:33613 [wandb_run.py:_console_start():1693] atexit reg
2021-11-09 22:39:50,432 INFO    MainThread:33613 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.REDIRECT
2021-11-09 22:39:50,432 INFO    MainThread:33613 [wandb_run.py:_redirect():1572] Redirecting console.
2021-11-09 22:39:50,434 INFO    MainThread:33613 [wandb_run.py:_redirect():1628] Redirects installed.
2021-11-09 22:39:50,434 INFO    MainThread:33613 [wandb_init.py:init():603] run started, returning control to user process
2021-11-09 22:39:52,833 INFO    MainThread:33613 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 1
2021-11-09 22:39:52,833 INFO    MainThread:33613 [wandb_run.py:_restore():1635] restore
2021-11-09 22:39:54,949 INFO    MainThread:33613 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 814
  total_bytes: 814
}

2021-11-09 22:39:55,320 INFO    MainThread:33613 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 814
  total_bytes: 814
}

2021-11-09 22:39:55,432 INFO    MainThread:33613 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 814
  total_bytes: 2252
}

2021-11-09 22:39:55,534 INFO    MainThread:33613 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 814
  total_bytes: 2252
}

2021-11-09 22:39:55,637 INFO    MainThread:33613 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 814
  total_bytes: 2252
}

2021-11-09 22:39:55,740 INFO    MainThread:33613 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 814
  total_bytes: 2252
}

2021-11-09 22:39:55,843 INFO    MainThread:33613 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 814
  total_bytes: 2252
}

2021-11-09 22:39:55,946 INFO    MainThread:33613 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2252
  total_bytes: 2252
}

2021-11-09 22:39:56,049 INFO    MainThread:33613 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2252
  total_bytes: 2252
}

2021-11-09 22:39:56,153 INFO    MainThread:33613 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2252
  total_bytes: 2252
}

2021-11-09 22:39:56,256 INFO    MainThread:33613 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2252
  total_bytes: 2252
}

2021-11-09 22:39:56,359 INFO    MainThread:33613 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2252
  total_bytes: 2252
}

2021-11-09 22:39:56,462 INFO    MainThread:33613 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2252
  total_bytes: 2252
}

2021-11-09 22:39:56,637 INFO    MainThread:33613 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2252
  total_bytes: 2252
}

2021-11-09 22:40:01,738 INFO    MainThread:33613 [wandb_run.py:_wait_for_finish():1793] got exit ret: None
2021-11-09 22:40:02,317 INFO    MainThread:33613 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2252
  total_bytes: 2252
}
local_info {
}

2021-11-09 22:40:03,416 INFO    MainThread:33613 [wandb_run.py:_append_files():2061] logging synced files
